### PR TITLE
Stripping out GCS bucket

### PR DESCRIPTION
**Pull request by mclacore**

### CODE REVIEW

File: massdriver.yaml [hunk 1]
```diff
@@ -3,29 +3,24 @@ name: gcp-gcs-bucket-static-assets
...
+    bucket:
+      type: object
+      title: Bucket
+      required:
+        - force_destroy
+      properties:
+        force_destroy:
+          title: Enable force destroy of bucket
+          description: When enabled, the bucket and all objects within it are destroyed when the bundle is decommissioned.
+          default: true
```
Comment (chrisghill) [inline massdriver.yaml:22]:
```suggestion
          default: false
```
Probably should default this to false.

File: src/_artifacts.tf [hunk 2]
```diff
@@ -1,7 +1,6 @@
 locals {
   data_infrastructure = {
     id  = google_storage_bucket.main.name
-    url = local.full_url
   }
   data_security = {
     iam = {
```
Comment (chrisghill) [inline src/_artifacts.tf:6]: I think the `read_write` role needs a different `role`. Check the slack convo...I think I recommended the proper role.

File: src/main.tf [hunk 3]
```diff
@@ -2,48 +2,12 @@ resource "google_storage_bucket" "main" {
...
-    response_header = ["*"]
-    max_age_seconds = 3600
-  }
+  force_destroy               = var.bucket.force_destroy
+  # Enables IAM permissions to bucket, revokes ACL permissions
+  uniform_bucket_level_access = true
 }
 
 resource "google_compute_backend_bucket" "main" {
   name        = var.md_metadata.name_prefix
```
Comment (chrisghill) [inline src/main.tf:11]: I don't think we need this resource: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_backend_bucket

**Review by chrisghill — COMMENTED** [review]
Some comments. Let's find a time to sync on this tomorrow.

**Comment by mclacore** [comment]
Closing PR because we are shifting to make a new bundle instead for gcp-gcs-app-assets

**Pull request closed without merging**